src/main/java/dev/isxander/adaptivetooltips/mixins/GameRendererMixin.java

```diff
@@ -10,7 +10,7 @@
 @Mixin(GameRenderer.class)
 public class GameRendererMixin {
     @Inject(method = "render", at = @At("RETURN"))
-    private void checkTooltipRendered(float tickDelta, long startTime, boolean tick, CallbackInfo ci) {
+    private void checkTooltipRendered(CallbackInfo ci) {
```

Comment:
```suggestion
    private void checkTooltipRendered(DeltaTracker tracker, boolean tick, CallbackInfo ci) {
```
Why remove parameters of the original method when there was only a change in parameters.
DeltaTracker would then need to be imported as such:
```java
import net.minecraft.client.DeltaTracker;
```

Reply:
If the parameters aren't used they can be removed. This improves compatibility between versions.

Reply:
Ok, that's cool, I didn't know it was possible.